### ISSUE TITLE
Update neighbourhoods in Hamburg, Germany

### ISSUE THREAD

**Issue by soulsurfer4** [whosonfirst-data/whosonfirst-data#1816]
The description of "https://spelunker.whosonfirst.org/id/85899703/" returns "Allermöhe" but correct would be "Eimsbüttel". 
Please see: https://www.openstreetmap.org/search?query=Scheideweg%20Hamburg 
show: Scheideweg, Hoheluft-West, Eimsbüttel, Hamburg, 20253, Germany

There are many other mislead data sets.

**Comment by nvkelso** [comment]
Hi @soulsurfer4. Thanks for the report. Do you have a good source for this missing information, aside from OpenStreetMap?

**Comment by justinelliotmeyers** [comment]
https://geoportal-hamburg.de/geo-online/ help?

**Comment by Soulsurfering** [comment]
Hi @nvkelso No, sorry. I don't have any other data source to provide detailed information.

**Comment by stepps00** [comment]
I can dig into this a bit more, but I think the issue here is less that the geometries are incorrect, and more that the  [Neu Lokstedt](https://spelunker.whosonfirst.org/id/85899703/) record has incorrect name translations for "Allermöhe". These translations are coming from a bunk wikidata concordance in the record.

**Comment by pathmapper** [comment]
Here is the official data from the City of Hamburg (Freie und Hansestadt Hamburg, Landesbetrieb Geoinformation und Vermessung):

http://suche.transparenz.hamburg.de/dataset/alkis-verwaltungsgrenzen-hamburg
e.g. this GML file:
http://archiv.transparenz.hamburg.de/hmbtgarchive/HMDK/alkisverwaltungsgrenzen_hh_2016-03-07_3989_snap_1_10818_snap_1.ZIP

License: 
Data licence Germany – attribution – version 2.0
https://www.govdata.de/dl-de/by-2-0

**Comment by stepps00** [comment]
@soulsurfer4 - I've updated name properties in the "Neu Lokstedt" record you linked to in your initial comment in the PR (above). This will fix some issues you're seeing with the "Allermöhe" name, but there is additional work to do in WOF to update neighbourhoods and other placetypes in Hamburg.

I've updated the issue title so we can track additional updates here. There are two comments above that have great sources, so we should have enough to work with.
